spec/classes/init_spec.rb

```diff
@@ -65,8 +65,7 @@ def remove_data
         }
 
         if ['RedHat', 'CentOS'].include?(facts[:os][:name])
-          if ['6','7'].include?(facts[:os][:release][:major])
-
+          if facts[:os][:release][:major] == '7'
```

Comment:
Should this be `if ['7','8'].include?(facts[:os][:release][:major])`.  EL8 support was added a while ago.  Do not understand why missing from this list?